benchbuild/utils/schedule_tree.py

```diff
@@ -89,8 +93,8 @@ def indent(self, level=0, idt=' '):
 SEQ_ELEM_LIST = p.delimitedList(ROOT)
 SEQUENCE = KW_SEQUENCE + ":" + "[" + p.Group(p.delimitedList(ROOT)) + "]"
 CHILD = KW_CHILD + ":" + ROOT
-CHILD_NODE << (CHILD | COINCIDENT | DOMAIN | EXTENSION | FILTER | MARK
-               | OPTIONS | PERMUTABLE | SCHEDULE | SEQUENCE)
+CHILD_NODE << (CHILD | COINCIDENT | DOMAIN | EXTENSION | FILTER | MARK |
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)